app/renderer/components/preferences/helpfulHints.js

```diff
@@ -40,11 +40,15 @@ const styles = StyleSheet.create({
   },
 
   helpfulHints: {
-    width: '100%',
-    height: '100%',
     cursor: 'default',
-    padding: '30px 0',
-    marginTop: '20px'
+    padding: '20px 0 0',
+    overflowY: 'auto',
```

Comment:
adding sync and plugins tabs made menu taller, so helpfulHints got cut.

For small displays it adds a scrollbar to address that